packages/marketing-modal/src/MarketingModal.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import Button from '@leafygreen-ui/button';
+import { Link } from '@leafygreen-ui/typography';
+import Modal from '@leafygreen-ui/modal';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const titleStyle = css`
+  color: ${uiColors.gray.dark2};
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 25px;
+
+  margin-bottom: 10px;
+`;
+
+const baseModalStyle = css`
+  width: 600px;
+  padding: initial;
+  overflow: scroll;
+`;
+
+const baseCoverStyle = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  & > * {
+    display: block;
+  }
+`;
+
+const defaultCoverStyle = css`
+  padding-top: 20px;
+  padding-bottom: 8px;
+`;
+
+const coverCoverStyle = css`
+  padding-bottom: 24px;
+
+  & > * {
+    width: 100%;
+  }
+`;
+
+const contentStyle = css`
+  color: ${uiColors.gray.dark1};
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 24px;
+  text-align: center;
+
+  padding: 0 92px;
+  padding-bottom: 24px;
+`;
+
+const footerContentStyle = css`
+  line-height: 24px;
+  padding-bottom: 50px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  & > * {
+    padding: 4px;
+  }
+`;
+
+interface MarketingModalProps {
+  title: string;
+  cover: JSX.IntrinsicElements['img'];
+  coverStyle?: 'default' | 'cover';
```

Comment:
If I'm understanding this correctly, this is to switch between full-bleed background images, and centered images correct?

I would consider:
- Using a `variant` prop to be consistent with other components in LeafyGreen. This also lets us make changes unrelated to the image itself in the modal based on the variant in the future if we need.
- If not, I might call this `imageStyle`, and having `fill` and `center` be the variants here.

Reply:
>If I'm understanding this correctly, this is to switch between full-bleed background images, and centered images correct?

Right. I think I explicitly chose not to use `variant` because I don't have a sense of what other ways this component will be customizable in the future and it may be the case we'll want to allow consumers to choose different combinations of those props. My thinking is that in the future `variant` can set the default configuration of multiple props and then individual props like `imageStyle` can override them, or we could make them mutually exclusive, i.e. one must either provide the `variant` prop or explicit set of more granular props. So for now I think I'll go with `imageStyle`.
> and having `fill` and `center` be the variants here.

should the prop continue to be optional and default to `center`?